.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: CSharp
+install: 
+ - echo "deb http://download.mono-project.com/repo/debian nightly main" | sudo tee /etc/apt/sources.list.d/mono-nightly.list
```

Comment:
Do you mean `sudo add-apt-repository`?
________________________________
From: David Fowler<mailto:notifications@github.com>
Sent: ‎1/‎06/‎2015 14:03
To: JabbR/JabbRv2<mailto:JabbRv2@noreply.github.com>
Cc: Henrik Andersson<mailto:henrik.a.andersson@outlook.com>
Subject: Re: [JabbRv2] Get Travis-CI build going (#9)

> @@ -0,0 +1,8 @@
> +language: CSharp
> +install:
> + - echo "deb http://download.mono-project.com/repo/debian nightly main" | sudo tee /etc/apt/sources.list.d/mono-nightly.list

There's a more first class way to do that

---
Reply to this email directly or view it on GitHub:
https://github.com/JabbR/JabbRv2/pull/9/files#r31399791